wpf/HelloWorld-WithLinkedFiles/HelloWorldCore/HelloWorldCore.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Wpf">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
This approach does not work if you want have both projects functional. Looks like it's also importing the generated code from NetFx/obj folder. 

We could Exclude obj, but it does not look correct to me. Also I dont like to do not see the linked files in the Core project.

I agree with globbing in a per folder basis, but not for everthing since it could have other problems for real solutions.

Reply:
You should see the linked files in the core project with my suggestion.  Let's take a look together if that's not working.

Personally, I think it's best to start with a template that includes all of the source files from the linked project.  That way you can get started even on complex projects with lots of files without having to muck about with your project file.  The following should exclude generated .cs files from the obj folder and I think looks a bit better than my first suggestion:

```xml
<PropertyGroup>
  <LinkedProjectPath>..\HelloWorldNetFx</LinkedProjectPath>
</PropertyGroup>
<ItemGroup>
  <ApplicationDefinition Include="$(LinkedProjectPath)\App.xaml" />

  <Compile Include="$(LinkedProjectPath)\**\*.cs" />
  <Compile Remove="$(LinkedProjectPath)\Properties\AssemblyInfo.cs" />
  <Compile Remove="$(LinkedProjectPath)\obj\**\*" />

  <Page Include="$(LinkedProjectPath)\**\*.xaml" />
  <Page Remove="@(ApplicationDefinition)" />
</ItemGroup>
```

Reply:
>  That way you can get started even on complex projects with lots of files without having to muck about with your project file. 

I like the idea of having a template that's easy to apply to *any* project, but I guess we need to add more file types such as resources, etc.. 

I can see the `.cs` files but not `.xaml` files:

![image](https://user-images.githubusercontent.com/14916339/47168784-9cb83980-d2b6-11e8-89bd-565bb7378387.png)

I'd suggest to try the *template* with additional projects to make sure we capture the most common file types before making it as a general recommendation.